rep-2010.rst

```diff
@@ -220,7 +219,34 @@ And the definitions for ``example_interfaces/srv/AddTwoInts_Reponse`` is,
 Configuration
 -------------
 
-TODO: Describe what configuration options there are and how we plan to expose them (e.g. with ROS parameters and/or environment variables).
+Configuration of service introspection features will be done through ROS parameters.
+
+.. code-block:: yaml
+
+   /NODE_NAME:
+      ros__parameters:
+         
+         # Indicates if CLIENT_SENT and CLIENT_RECEIVED events will be published
+         SERVICE_NAME/client_events_enabled: bool
```

Comment:
I'm not sure if we want to include the service name in the parameter names, then we'll end up with N * Sn * 6 parameters, where N is number of nodes and Sn is number of services for each node. Though this gives the user fine grain control over which services provided by a node will emit events, I'm not sure if it's worth the added complexity. I imagine the more common use case for the feature will be to turn it on for all services for a node (or all nodes in the system). We could go so far as having a single parameter per node, but I think a good compromise are the existing 6 parameters that give the user some control over the number (and size of) event messages.

Reply:
See [83f0841](https://github.com/jacobperron/rep/pull/2/commits/83f084112210ca4a6bb279f11ef93a5311b783db)

Reply:
@deepanshubansal01 and I had a discussion about this and figured it would be nice in the event that a node offers multiple large services. We can wait on some other feedback on this, though I agree that enabling/disabling for all services in a node should cover most cases.